mission_control-jobs.gemspec

```diff
@@ -16,7 +16,11 @@ Gem::Specification.new do |spec|
     Dir["{app,config,db,lib}/**/*", "MIT-LICENSE", "Rakefile", "README.md"]
   end
 
-  spec.add_dependency "rails", ">= 7.1"
+  rails_version = ">= 7.1"
+  spec.add_dependency "activerecord", rails_version
+  spec.add_dependency "activejob", rails_version
+  spec.add_dependency "railties", rails_version
+  spec.add_dependency "propshaft"
```

Comment:
Why does this add this new dependency?

Reply:
@rosa I can't remember, and it's odd, i don't even use propshaft myself.

Thanks for merging #194! ☺️